src/extensions/default/CSSCodeHints/CSSProperties.json

```diff
@@ -209,6 +209,7 @@
     "text-emphasis-position":      {"values": ["above", "below", "left", "right"]},
     "text-emphasis-style":         {"values": ["circle", "dot", "double-circle", "filled", "none", "open", "sesame", "triangle"]},
     "text-indent":                 {"values": ["inherit"]},
+    "text-justify":                {"values": ["auto", "none", "inter-word", "inter-character"]},
```

Comment:
we have "inherit" as an option for most of the properties. maybe we can add here as well since it is applicable.